examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>TYPO: randomely -&gt; randomly</p><p>Maybe we can also add that the offline phase exists to speed up the online phase (you actually pointed this out).</p><p>But, we can also generate Beaver Triples during the online phase - but this would impact the <code>online</code>  performance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Ok thanks, I'll make that clearer!</p>